cmake/compile_tests/pthread.cpp

```diff
@@ -4,7 +4,7 @@ void* kokkos_test(void* args) { return args; }
 
 int main(void) {
   pthread_t thread;
-  pthread_create(&thread, nullptr, kokkos_test, nullptr);
-  pthread_join(thread, nullptr);
+  pthread_create(&thread, NULL, kokkos_test, NULL);
+  pthread_join(thread, NULL);
```

Comment:
I would rather add a comment here than above that we cannot convert `NULL` to `nullptr` since this might require additional C++ standard flags.

Reply:
Added comment. Fixed whitespace.